src/main/java/com/github/wovnio/wovnjava/Api.java

```diff
@@ -155,6 +160,10 @@ private URL getApiUrl(String lang, String body) throws UnsupportedEncodingExcept
         appendValue(sb, lang);
         appendValue(sb, "&version=wovnjava_");
         appendValue(sb, Settings.VERSION);
+        if (this.requestOptions.getCacheDisableMode() || this.requestOptions.getDebugMode()) {
+            appendValue(sb, "&timestamp=");
+            appendValue(sb, String.valueOf(System.currentTimeMillis()));
```

Comment:
I think you should use `appendKeyValue()` instead of the two lines.
If not, looking at `appendKeyValue()`, it seems `appendValue(sb, "&timestamp=");` should be replaced by `append(sb, "&timestamp=");`?

Reply:
`appendValue` is used because it encodes the appended string, and I believe the cache key needs to be fully encoded, as done here.

`appendKeyValue` is used in other places to encode only the value, not the key.

Reply:
I see, we probably can do better method naming (not here tho).